### PR TITLE
Added the custom delete message functionality with params

### PR DESCRIPTION
**Pull request by mangei**

### CODE REVIEW

File: src/main/javascript/directives/cat-confirm-click.js [hunk 1]
```diff
@@ -3,18 +3,34 @@
...
-    .directive('catConfirmClick', function CatConfirmClickDirective() {
+    .directive('catConfirmClick', function CatConfirmClickDirective(catI18nService) {
         return {
             restrict: 'A',
             link: function CatConfirmClickLink(scope, element, attr) {
                 var msg = attr.catConfirmClick || 'Are you sure?';
                 var clickAction = attr.catOnConfirm;
+                var params = undefined;
+                if (!!attr.catI18nParams) {
+                    params = JSON.parse(attr.catI18nParams);
```
Comment (tscheinecker) [inline src/main/javascript/directives/cat-confirm-click.js]: i think what you want to do is parse it as an angular expression
see https://docs.angularjs.org/api/ng/service/$parse

Comment (tpinetz) [inline src/main/javascript/directives/cat-confirm-click.js]: Yes I will fix this, when I have the time.

**Comment by tscheinecker** [comment]
ping

**Comment by tpinetz** [comment]
This PR can be closed. The real one is: https://github.com/Catalysts/cat-angular/pull/60

**Pull request closed without merging**